draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -130,7 +138,88 @@ This document uses "service" and "workload" interchangeably. Otherwise, all term
 
 # Application Level Service To Service Authentication {#app-level}
 
-## The WIMSE ID Token
+## The Workload Identity Token {#to-wit}
+
+The Workload Identity Token (WIT) is a JWS {{RFC7515}} signed JWT {{RFC7519}} that represents the identity of a workload.
+It is issued by the Identity Server and binds a public key to the workload identity.
+A WIT MUST contain the following:
+
+* in the JOSE header:
+    * `alg`: An identifier for a JWS asymmetric digital signature algorithm
+     (registered algorithm identifiers are listed in the IANA JOSE Algorithms registry {{IANA.JOSE.ALGS}}). The value `none` MUST NOT be used.
+    * `typ`: the WIT is explicitly typed, as recommended in {{Section 3.11 of RFC8725}}, using the `wimse-id+jwt` media type.
+* in the JWT claims:
+    * `iss`: The issuer of the token, which is the Identity Server, represented by a URI.
+    * `sub`: The subject of the token, which is the identity of the workload, represented by a URI.
+    * `exp`: The expiration time of the token (as defined in {{Section 4.1.4 of RFC7519}}).
+      WITs should be refreshed regularly, e.g. on the order of hours.
+    * `jti`: A unique identifier for the token.
+    * `cnf`: A confirmation claim containing the public key of the workload using the `jwk` member as defined in {{Section 3.2 of RFC7800}}.
+     The workload MUST prove possession of the corresponding private key when presenting the WIT to another party.
```

Comment:
I think this statement is not useful because most developers don't know what it means and whether they're even compliant with this MUST. Moreover, the next 3 sections will each include a (sometimes implicit) proof of possession.

Reply:
I'd like to keep it to call attention to the need to do something other than just accept this as a bearer token. RFC7800 is not super well known and is a little opaque. And not even clear (to me anyway) if the presence of the cnf requires PoP. 

What about mentioning or referencing the next sections to give context about how one would be compliant with this MUST?

Reply:
Yes, please refer to the next sections. But maybe also reverse the sense of the MUST: the WIT MUST NOT be used as a bearer token. Instead, it should be used in conjunction with one of the methods in sections foo, bar, and baz.

Reply:
will do